pkg/image/charts.go

```diff
@@ -71,7 +71,7 @@ func (c Charts) FetchImages(imagesSet map[string]map[string]struct{}) error {
 		}
 		// Append the remaining versions of the chart if the chart exists in the chartsToCheckConstraints map
 		// and the given Rancher version satisfies the chart's Rancher version constraint annotation.
-		chartName := versions[0].Metadata.Name
+		chartName := versions[0].Name
```

Comment:
What's the benefit of using `.Name` over `.Metadata.Name`? Are they always the same?

Reply:
Linter was complaining that using `.Metadata` was unnecessary.